pilot/pkg/bootstrap/server.go

```diff
@@ -772,6 +771,10 @@ func (s *Server) initDiscoveryService(args *PilotArgs) error {
 		MixerSAN:         s.mixerSAN,
 	}
 
+	environment.UseRDS = os.Getenv("USE_RDS") == "true"
```

Comment:
You can change it to "DISABLE_RDS" - I assume RDS will be the default, we'll keep in-line while testing.